src/annotator/config/index.js

```diff
@@ -28,6 +29,7 @@ export default function configFrom(window_) {
     onLayoutChange: settings.hostPageSetting('onLayoutChange'),
     openSidebar: settings.hostPageSetting('openSidebar', {
       allowInBrowserExt: true,
+      coerce: toBoolean,
```

Comment:
There may be other values we want to force as a boolean, but only this one was needed for this ticket... and only because we only assume things coming from via are a string. (not all these settings from from via)